pyrasterframes/src/main/python/pyrasterframes/rf_ipython.py

```diff
@@ -0,0 +1,110 @@
+#
+# This software is licensed under the Apache 2 license, quoted below.
+#
+# Copyright 2019 Astraea, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+# [http://www.apache.org/licenses/LICENSE-2.0]
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+import pyrasterframes.rf_types
+
+
+def tile_to_png(tile):
+    """ Provide image of Tile."""
+    if tile.cells is None:
+        return None
+
+    import io
+    from matplotlib.backends.backend_agg import FigureCanvasAgg as FigureCanvas
+    from matplotlib.figure import Figure
+
+    # Set up matplotlib objects
+    [width, height] = tile.dimensions()
+    dpi = 300
+    nominal_size = 5.5  # approx full size for a 256x256 tile
+    fig = Figure(figsize=(nominal_size * width / dpi,   nominal_size * height / dpi))
+    canvas = FigureCanvas(fig)
+    axis = fig.add_subplot(1, 1, 1)
+
+    axis.imshow(tile.cells)
+    axis.set_aspect('equal')
+    axis.xaxis.set_ticks([])
+    axis.yaxis.set_ticks([])
+
+    axis.set_title('{}, {})'.format(tile.dimensions(), tile.cell_type.__repr__()))  # compact metadata as title
+
+    with io.BytesIO() as output:
+        canvas.print_png(output)
+        return output.getvalue()
```

Comment:
If we were still in DataFrame space, I'd push to do this work on the JVM side, adding a method to `PyRFContext` that essentially implements this:

https://github.com/locationtech/rasterframes/blob/136e222afbb3d1beff1d95e5d8587a23f5ff14b2/core/src/test/scala/examples/Masking.scala#L51-L54

(or [this](https://github.com/locationtech/rasterframes/blob/136e222afbb3d1beff1d95e5d8587a23f5ff14b2/docs/src/main/tut/exporting-rasterframes.md#exporting-a-raster))

The above just seems like an overly involved process to get a PNG representation when we've already got code to do it. However, I get that the `tile` parameter at this point has been already pulled out of the Spark context, and is fully represented in the Python VM, and pumping it back over the Py4J gateway is silly. So I guess I'll have to live with it (but not like it) ;-).